### PR TITLE
Allows vox to drink synthanol without throwing up

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

This PR changes the synthanol check from `if(!M.isSynthetic())` to `if(!(M.dna.species.reagent_tag & PROCESS_SYN))` Meaning that anyone that can process synthetic chemicals can drink it without throwing up

## Why It's Good For The Game

Allows vox to drink synthanol, like the other synthetic chemicals they can process without throwing up, as well as any other person var edited with the ability to process synthetic chems as well

## Changelog
:cl:
fix: Allows anyone with the PROCESS_SYN to drink synthanol and it's sub types without throwing up
/:cl:

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Gross

**Comment by Carthusia** [comment]
I recall me and another vox the day that update went live, we looted like 10-20 cans of beepskies premium synthahol, and drank every single can, throwing up all over ourselves the entire time, but never getting drunk.

It was, and i quote "Total bullshit" and a "Complete waste of ti BLUREGHHHHHHHHH"

File: code/modules/reagents/chemistry/reagents/alcohol.dm [hunk 1]
```diff
@@ -1190,15 +1190,17 @@
 	taste_description = "motor oil"
 
 /datum/reagent/consumable/ethanol/synthanol/on_mob_life(mob/living/M)
-	if(!M.isSynthetic())
-		holder.remove_reagent(id, 3.6) //gets removed from organics very fast
-		if(prob(25))
-			holder.remove_reagent(id, 15)
-			M.fakevomit()
-	return ..()
+    if(!(M.dna.species.reagent_tag & PROCESS_SYN))
```
Comment (Fox-McCloud) [inline code/modules/reagents/chemistry/reagents/alcohol.dm]: I should shoot you for using spaces instead of tabs.

**Comment by Fox-McCloud** [comment]
Spaces. Spaces everywhere still.

**Comment by KasparoVy** [comment]
Pounding shots of synthanol like a real bad bird

**Comment by Carthusia** [comment]
Skreeee!

Absinthe, mixed with a shot of Uplink, the drink of champions

**Comment by Fox-McCloud** [comment]
More of a fix than a tweak; this was my intention from the start; just an oversight on my part.

**Comment by KasparoVy** [comment]
Nice addition, glad to see this merged

**Pull request merged**